config/checkstyle_checks.xml

```diff
@@ -557,6 +557,12 @@
       <property name="file" value="${checkstyle.importcontroltest.file}"/>
       <property name="path" value="^.*[\\/]src[\\/]test[\\/].*$"/>
     </module>
+    <!-- until https://github.com/checkstyle/checkstyle/issues/9142 -->
+    <module name="ImportControl">
+      <property name="id" value="ImportControlAssertions"/>
+      <property name="file" value="${checkstyle.importcontrolassertions.file}"/>
```

Comment:
Why do we need a whole new import control file? Why can't we use the existing `importcontroltest`?

Reply:
I did it just to make it more clear and explicit. We anyway have to remove it after a month once I complete the migration.

Reply:
There might be no good reason to have separate file as content of it is not big.
it is just ` <disallow pkg="org.junit.jupiter.api.Assertions"/>` so you can copy it to main import control and it will work the same.
in addition you will not need to update other repositories to have such property.

Reply:
>  We anyway have to remove it after a month once I complete the migration.

This is another reason to avoid going through the hassle of adding a file just to eventually remove it.

Reply:
done